### DISCUSSION TITLE
No interactive mode

### DISCUSSION THREAD

**Discussion by vaelu** [iterate-ch/cyberduck#14923] [Q&A]
I use duck.sh in a bash script and use `$?` for determining if the command was successful or not. The problem is that when for example it does not find the container, it gets interactive and asks "Continue (y/n)". When I press `no`, it returns the correct error status and fails, but when I press `yes`, it just continues and returns 0.

Is this expected behavior? Is there any no-interactive mode?

**Comment by dkocher** [comment]
>  -y,--assumeyes                           Assume yes for all prompts

Use this flag to disable interactive mode.

**Reply by vaelu** [reply]
Yes, I know this flag. The problem is that it still returns 0, although the transfer is incomplete.

**Comment by vaelu** [comment]
To clarify: An incomplete transfer returns 0 as exit-code.

```bash
File not found. /testing. Please contact your web hosting service provider for assistance.
Continue (y/n): y
Transfer incomplete. test.txt…
root@linux: $ echo "$?"
0
```
